pkg/annotation/annotation.go

```diff
@@ -0,0 +1,199 @@
+package annotation
+
+import (
+	"bufio"
+	"io"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var (
+	// IgnoreRegexp finds the gocover ignore pattern.
+	// Three kind ignore pattern are supported:
+	//   //+gocover:ignore:all
+	//   //+gocover:ignore:block
+	//   //+gocover:ignore:{number}
+	//
+	// - `//+gocover:ignore:all`
+	//   will ignore the whole file, the file won't be used to calculate coverage.
+	//
+	// - `//+gocover:ignore:block`
```

Comment:
nit: can we link to the language spec: https://go.dev/ref/spec#Blocks to clarify what's the block here?

Reply:
I also feel the block definition here is different than the one in the go language spec. Can we consider using the same definition as the lang spec? I think by doing this, we can solve the question in L50